dotCMS/src/main/java/com/dotcms/content/model/version/ToCurrentVersionConverter.java

```diff
@@ -0,0 +1,52 @@
+package com.dotcms.content.model.version;
+
+import com.dotmarketing.util.Logger;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.jonpeterson.jackson.module.versioning.VersionedModelConverter;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ *
+ */
+public class ToCurrentVersionConverter implements VersionedModelConverter {
```

Comment:
shouldn't be called ToVersion2Converter?

Reply:
nope, because this class is intended to be used for any future migration